htdocs/theme/eldy/global.inc.php

```diff
@@ -7994,6 +7994,11 @@
 	}
 }
 
+/* for hide a select */
+.hideSelect {
```

Comment:
There is already a class called "hideobject" with the display: none. 
Can you use this one instead of adding a new one into the CSS ?